nengo/spa/actions.py

```diff
@@ -154,7 +154,11 @@ class Actions(object):
     def __init__(self, *args, **kwargs):
         self.actions = None
         self.args = args
-        self.kwargs = kwargs
+        self.kwargs = OrderedDict(kwargs)
```

Comment:
I think I'm going to got with @hunse suggestion of sorting it alphabetically before processing. That way, we have deterministic behaviour can tested and if for some bizarre reason the user does want an oddly specific order, they can just prepend their `kwargs` with an index.